src/include/function/arithmetic/arithmetic_functions.h

```diff
@@ -89,20 +100,35 @@ struct Abs {
     }
 };
 
+template<>
+inline void Abs::operation(common::int128_t& input, common::int128_t& result) {
```

Comment:
Since you have already implemented operations for int128_t, do you still need to specialize hugeint?

Reply:
yes here, since for abs we use c++'s` std::abs` which does not support int128, so we need to specialize hugeint, but for other functions like add, sub we don't need to specialize.